docs/ibmcloud_help.md

```diff
@@ -3,12 +3,101 @@
 **Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
 
 - [IBM Cloud](#ibm-cloud)
+  - [Inspect files `cluster.yaml` and `machines.yaml`.](#inspect-files-clusteryaml-and-machinesyaml)
   - [How to set correct OS reference code in `machines.yaml`?](#how-to-set-correct-os-reference-code-in-machinesyaml)
-  - [how to set `dataCenter` value in `machines.yaml`?](#how-to-set-datacenter-value-in-machinesyaml)
+  - [How to set `dataCenter` value in `machines.yaml`?](#how-to-set-datacenter-value-in-machinesyaml)
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
 # IBM Cloud
+## Inspect files `cluster.yaml` and `machines.yaml`.
+
+cluster.yaml
+
+Use the default cluser name test1 or replace with a new cluser name.
+```
+apiVersion: "cluster.k8s.io/v1alpha1"
+kind: Cluster
+metadata:
+  name: test1                                           # cluster name
+spec:
+    clusterNetwork:
+        services:
+            cidrBlocks: ["10.96.0.0/12"]
+        pods:
+            cidrBlocks: ["192.168.0.0/16"]
+        serviceDomain: "cluster.local"
+    providerSpec:
+      value:
+        apiVersion: "ibmcloudclusterproviderconfig/v1alpha1"
+        kind: "IBMCloudClusterProviderSpec"
+```
+
+machines.yaml
+
+1. Update the values of `domain, dataCenter, osReferenceCode`.
+2. Make sure use the same cluster name in labels.
+3. Use valid SSH Key name and user name.
+```
+apiVersion: "cluster.k8s.io/v1alpha1"
+kind: MachineList
+items:
+- apiVersion: "cluster.k8s.io/v1alpha1"
+  kind: Machine
+  metadata:
+    generateName: ibmcloud-master-                         # Prefix of device name
```

Comment:
Shall we just add such comments directly in `machines.yaml.template`? It is easy to read by end user. 

And for me, not all comments are is necessary.
E.g. need explaination: `generateName: ibmcloud-master- ` will be VM instance name prefix
E.g. makes less sense: `cluster.k8s.io/cluster-name: "test1"                 # cluser name`, cluster name explaining cluster name makes less sense.

Reply:
File achines.yaml.template is ok, and remind user to do a inspection for yaml files in ReadMe.md. What do you think?